### DISCUSSION TITLE
Ventura-BigSur-Monterey- OC-0.8.4 Release- Infinix

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#6] [Discuss about the release]
### **This EFI supports for VENTURA**
Installation Instructions: [Here](https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide#bios)

- Reworked EFI to work with <ins>Ventura beta 5</ins>. So, anyone can move to Ventura beta 5 or greater without any issue.
- Updated OpenCore version from 0.8.2 to 0.8.4
- Added PCI to Device-Properties.
- Fixed bluetooth in Ventura.
- Fixed Wifi in Ventura.
- Fixed device Heating issue also in Ventura.
- Increased Stability.
- Fixed Everything in Ventura Beta 5. Now You can use EFI for Ventura beta 5. Everything works like BigSur in Ventura.
- Re-worked ACPI  session.
- Donation will be helpful for future Updates: [Here](https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide#any-one-can-donate-me-or-buy-me-a-coffee-)
- Fixed NVM Express Management under PCI session.
- Optimised sleep issue & Lid issue. So, You can put your laptop for more long hours.
- Removed Unnecessary stuff in device properties.
- Fixed iCloud.
- Reworked with Power Management.
- Increased Battery Backup.

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/OC-8.4-infinix-inbook'>Ventura-BigSur-Monterey- OC-0.8.4 Release- Infinix</a>.</em>

**Comment by internetvuser** [comment]
hi can you please fix the apple logo changed by so many words at boot

**Reply by internetvuser** [reply]
and i want to ask why from the other released graphics 3072 mb and now 1536 mb

**Reply by devboloji** [reply]
@internetvuser Read post install below of readme when? And Understand what is thoses text here -> https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/discussions/2#discussioncomment-3221231

**Reply by devboloji** [reply]
> and i want to ask why from the other released graphics 3072 mb and now 1536 mb

@internetvuser This is due to Ventura is in beta. The performance is 19% low than Monterey, Tested. But this is berry smooth. This issue may fix when the Apple.Inc releases Ventura stable.

**Reply by internetvuser** [reply]
I use Monterey, not Ventura, thank you for the answer

**Comment by Causoares2003** [comment]
Hi, i have a question, I used your efi for Monterey on my Asus X515JA with the same cpu/gpu and it worked fine! Can you tell me please if I can use your SSDTs on my system?

**Reply by devboloji** [reply]
@Causoares2003 can you mention about your all device specs.. Like device full name, cpu, gpu, ram, storage, wifi-model, Bluetooth - model, and all stuff. I am happy 😊 that you successfully booted macOS with my EFI And 🎊 Congratulations.

**Reply by devboloji** [reply]
> Hi, i have a question, I used your efi for Monterey on my Asus X515JA with the same cpu/gpu and it worked fine! Can you tell me please if I can use your SSDTs on my system?

@Causoares2003 @internetvuser I asked him the EFI for Asus vivobook 15 X515JA , but he is asking me 30bucks [image-Proof](https://imgur.com/RmpFVlj) . But I am giving the EFI for free to Infinix, There is no result for my Very HardWork and I am giving the Finest EFI by spending Hours by fixing. In Buying new macbook, I made infinix to Macbook air. In-place of buying apple, Donate me to speed up work ->[Here](https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide#any-one-can-donate-me-or-buy-me-a-coffee-)
Even I don't have the device, I may give updates to Asus X515JA, if anyone donate.

**Reply by devboloji** [reply]
@Causoares2003 yes' you can use it. But tell me what modifications you made

**Comment by devboloji** [comment]
@Causoares2003 @internetvuser Join the Telegram group for any discussion of infinix, [Group](https://t.me/infinix_inbook_Updates)

**Comment by devboloji** [comment]
In 'PCI session' on system information, you see sd host controller - driver
not installed. But you can the micro USB slot, Its working fine. I have
already had a look but I forgot to mention it in my repository. Anyway
donation will be helpful. To donate go to readme of infinix hackintosh
github and buy me a coffee ☕.

**Comment by internetvuser** [comment]
hi can you please remap the usb and remap the brightness key on the next release?

**Reply by devboloji** [reply]
Mention you device specifications & tell me the issue about USB remapping clearly.

**Reply by internetvuser** [reply]
infinix inbook x1, i use hackintool and not all usb detected

**Reply by devboloji** [reply]
how many usbs do you have?

**Reply by internetvuser** [reply]
4

**Reply by devboloji** [reply]
> 4
> 

Mention every port details...

**Reply by devboloji** [reply]
@internetvuser If you are in Ventura beta and want to remap brightness keys, go to `System Preferences` -> `Keyboard` -> `keyboard shortcuts` -> go to `Display` tab. Change the keys as you prefer. Hope you found the answer.

**Comment by extasea** [comment]
Hi @devboloji , thanks for the hardwork to update the EFI regularly
Do you have virtual wallet for IDR for donation? I'm indonesian by the way

Does this EFI version works for dual booting windows for dual NVME SSD? as we know the Inboox X1 had 2 slots of NVME SSD

My problem with EFI 0.8.1 was the system cannot boot normally with 2 NVME SSD installed.

Thankyou,

**Reply by miftasoft** [reply]
Hi @extasea , You can donate with PayPal. I have dual boot with different NVMe SSD, macOS on Foresee 256 GB and Windows (NTFS) + Data (exFAT) + Time Machine Backup (AFPS) on ADATA SX6000PNP 1TB

**Reply by extasea** [reply]
ohh I see, there is a PayPal down there hehehe,, my bad

can you share the tutor how to setting the EFI set up for dual boot with different SSD. Thanks

**Reply by devboloji** [reply]
@extasea I am happy to hear that. read the post install 9 in readme and join [telegram group](https://t.me/infinix_inbook_discussion) of our infinix for help or issues.

**Reply by extasea** [reply]
@devboloji enjoys the coffee mate,

**Reply by devboloji** [reply]
@extasea Thank you so much, I take this for encoragement for the future updates.

**Comment by internetvuser** [comment]
the speaker in fornt of infinix logo is not working only the 2 speaker under the laptop working
